src/Tizen.NUI/src/public/WebView/WebView.cs

```diff
@@ -27,10 +27,9 @@ namespace Tizen.NUI
     /// <summary>
     /// WebView
     /// </summary>
-    [EditorBrowsable(EditorBrowsableState.Never)]
+    /// <since_tizen> 9 </since_tizen>
```

Comment:
Please keep [EditorBrowsable(EditorBrowsableState.Never)] in DevelNUI branch.
When this patch is merged, you can remove it on Master branch with ACR.